en_US/data-integration/data-bridge-cassa.md

```diff
@@ -0,0 +1,112 @@
+- Knowledge about [data bridge](./data-bridges.md)
+- Relevant Cassandra keyspace should be created before creating the data bridge.
+
+<!-- 列举功能或性能方面的亮点，如支持批处理、支持异步模式、双向数据桥接，链接到对应的功能介绍章节。 -->
+
+:::
+
+## Feature list
+
+- [Connection pool](./data-bridges.md#Connection pool)
+- [SQL preprocessing](./data-bridges.md#Prepared statement)
+
+<!--  Configuration parameters TODO 链接到配置手册对应配置章节。 -->
+
+## Quick starts
+<!-- 从安装测试所需步骤，如果有不同的用法增加章节介绍。 -->
+
+### Install Cassandra
+
+We use need to start simple Cassandra service via docker:
+
+```bash
+docker run --name cassa --rm -p 9042:9042 cassandra:3.11.14
+```
+
+### Create Keyspace and table
+
+Create a Keyspace named `mqtt`:
+```bash
+docker exec -it cassa cqlsh "-e CREATE KEYSPACE mqtt WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1}"
+```
+
+Create a table in Cassandra: `mqtt_msg`
+```bash
+docker exec -it cassa cqlsh "-e \
+    CREATE TABLE mqtt.mqtt_msg( \
+        msgid text, \
+        topic text, \
+        qos int,    \
+        payload text, \
+        arrived timestamp, \
+        PRIMARY KEY(msgid, topic));"
+```
+
+:::tip
+These keyspace and tables must be created before we create the data bridge to Cassandra.
+:::
+
+### Create a data bridge to Cassandra
+
+1. Go to EMQX Dashboard, click **Data Integration** -> **Data Bridge**.
+2. Click **Create** on the top right corner of the page.
+3. In the **Create Data Bridge** page, click to select **Cassandra**, and then click **Next**.
+4. Input a name for the data bridge. Note: It should be a combination of upper/lower case letters or numbers.
+5. Input the connection information. Input `127.0.0.1:9042` for the **Servers**, `mqtt` as the **Keyspace**, and leave others as default.
+6. Click the **Create** button to complete the data bridge creation.
+
+We have successfully created the data bridge to Cassandra, now we can continue to create rules to specify the data to be stored to Cassandra. 
+
+1. Go to EMQX Dashboard, click **Data Integration** -> **Rules**.
+2. Click **Create** on the top right corner of the page.
+3. Input `my_rule` as the rule ID, and set the rules in the **SQL Editor**. Here we want to forward the MQTT messages under topic `t/#` to Cassandra, we can use the SQL syntax below. Note: If you are testing with your SQL, please ensure you have included all required fields in the `SELECT` part. 
+
+  ```sql
+  SELECT 
+    *
+  FROM
+    "t/#"
+  ```
+
+4. Then click the **Add Action** button, select **Forwarding with Data Bridge** from the dropdown list and then select the data bridge we just created under **Data bridge**.  
+5. Then, click the **Add** button. 
+6. Then click the **Create** button to finish the setup. 
+
+Now we have successfully created the data bridge to Cassandra. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under topic `t/#`  are sent and saved to Cassandra after parsing by rule `my_rule`.
```

Comment:
```suggestion
After creating the data bridge to Cassandra. You can click **Data Integration** -> **Flows** to view the topology. It can be seen that the messages under topic `t/#`  are sent and saved to Cassandra after parsing by rule `my_rule`.
```